cv32e40x/tests/programs/corev-dv/corev_rand_arithmetic_base_test/.gitignore

```diff
@@ -1 +0,0 @@
-corev_*.*
```

Comment:
Thanks for deleting these.  They are redundant and lead to maintenance issues in the future.